src/fh-family-experience-web/Pages/FamilyHub.cshtml.cs

```diff
@@ -53,11 +59,16 @@ public async Task<IActionResult> OnPostAsync()
         }
 
         TempData["PostCode"] = Postcode;
-        PostcodeIOResponse = await _postcodeLookupService.GetPostcodeAsync(Postcode);
+        PostcodeIOResponse = await _postcodeLookupService.GetPostcodeAsync(Postcode!);
+
+        if (PostcodeIOResponse is null)
+            throw new Exception("Failed to get response from Service Directory API");
```

Comment:
Shouldn't this be something like...
'Failed to get response from Postcode Lookup service'?